maestro-client/src/main/java/maestro/Driver.kt

```diff
@@ -69,4 +69,6 @@ interface Driver {
     fun takeScreenshot(out: Sink)
 
     fun setLocation(latitude: Double, longitude: Double)
+
+    fun eraseAllText(charactersToErase: Int)
```

Comment:
It isn't really "all text" if you give it a number of characters. Shall we just call it `eraseText`?